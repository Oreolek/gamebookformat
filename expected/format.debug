BEGIN DEBUG OUTPUT
Book title: Format
Number of sections: 4
Introduction
 Adding an introduction to the gamebook here. This will create a section, but it will not be shuffled nor numbered with the gamebook sections below. 
Another Heading
 This starts another non-shuffled section. 


Adventure begins in section 1.
1 (start) -  This examples tests gamebook formatting, not so much game mechanics or references. Currently there is nothing here really. This section contains some tricky characters to quote, like } and { and " and ' and \. HTML will probably not like <div> or &boom;. There should be an image below as well. If something broke, turn to 2, otherwise turn to 3. [IMG]testimage.png[/IMG] 
2 (bad) -  Bad.
3 (good) -  Good! 
4 (dum) -  Sections tagged as dummy will not be visible in output at all. 
END DEBUG OUTPUT
