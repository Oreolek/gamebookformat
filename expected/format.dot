digraph gamebook {

 1->2
1->3
 
}
