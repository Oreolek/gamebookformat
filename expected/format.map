{"named2": 5, "named1": 6, "good": 3, "dum": 7, "start": 1, "bad": 2, "formatspecific": 4}