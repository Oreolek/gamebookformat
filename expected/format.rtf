{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Format
\b0\
\
\b \qc Introduction
\b0\



\ql  Adding an introduction to the gamebook here. This will create a section, but it will not be shuffled nor numbered with the gamebook sections below.  \
\
\
\b \qc Another Heading
\b0\



\ql  This starts another non-shuffled section.  (A footnote in Another Heading.)
  \
\
\b Adventure begins in section 1.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  This examples tests gamebook formatting, not so much game mechanics or references. Currently there is nothing here really. This section contains some tricky characters to quote, like \} and \{ and " and ' and \\. HTML will probably not like <div> or &boom;. You can make named references \b {\field{\*\fldinst  REF s6 \\h }{\fldrslt like this}}
\b0

 (just happens to be exactly the same syntax as in emacs org-mode btw). This book also includes a \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt format-specific section}}
\b0

. There should be an image below as well. If something broke, turn to \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
, otherwise turn to \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
. (Good footnote.)
 
  \
\

\b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  Bad. \
\

\b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  Good!  \
\

\b \qc {\*\bkmkstart s4}4{\*\bkmkend s4}
\b0\
\ql  This is the format-specific section for no other formats. Now you \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt lose}}
\b0

.  \
\

\b \qc {\*\bkmkstart s5}5{\*\bkmkend s5}
\b0\
\ql  Very good. \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt You win}}
\b0

. (Win footnote.)
  \
\

\b \qc {\*\bkmkstart s6}6{\*\bkmkend s6}
\b0\
\ql  This is where you should end up when you follow the named reference from the starting section. From here you can go to \b {\field{\*\fldinst  REF s5 \\h }{\fldrslt the second named section}}
\b0

 or to the \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt good end}}
\b0

.  \
\

}
