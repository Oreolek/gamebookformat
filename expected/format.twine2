<tw-storydata name="Format" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This examples tests gamebook formatting, not so much game mechanics or references. Currently there is nothing here really. This section contains some tricky characters to quote, like } and { and " and ' and \. HTML will probably not like  `<` div `>`  or  `&` boom;. You can make named references [[like this->6]] (just happens to be exactly the same syntax as in emacs org-mode btw). This book also includes a [[format-specific section->4]]. There should be an image below as well. If something broke, turn to [[2]], otherwise turn to [[3]]. (Good footnote.)
 
 </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> Bad.</tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> Good! </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the format-specific section for no other formats. Now you [[lose->2]]. </tw-passagedata>
<tw-passagedata pid="5" name="5" tags="" position="50,50"> Very good. [[You win->3]]. (Win footnote.)
 </tw-passagedata>
<tw-passagedata pid="6" name="6" tags="" position="60,60"> This is where you should end up when you follow the named reference from the starting section. From here you can go to [[the second named section->5]] or to the [[good end->3]]. </tw-passagedata>
</tw-storydata>
