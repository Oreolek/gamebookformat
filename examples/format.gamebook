title = Format
* 1 start
This examples tests gamebook formatting, not so much game mechanics or
references. Currently there is nothing here really.
