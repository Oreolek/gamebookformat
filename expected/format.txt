    Format

Introduction
 Adding an introduction to the gamebook here. This will create a section, but it will not be shuffled nor numbered with the gamebook sections below. 

Another Heading
 This starts another non-shuffled section.  (A footnote in Another Heading.)
 


Adventure begins in section 1.
1
 This examples tests gamebook formatting, not so much game mechanics or references. Currently there is nothing here really. This section contains some tricky characters to quote, like } and { and " and ' and \. HTML will probably not like <div> or &boom;. You can make named references like this (5) (just happens to be exactly the same syntax as in emacs org-mode btw). There should be an image below as well. If something broke, turn to 2, otherwise turn to 3. (Good footnote.)
 
 

2
 Bad.

3
 Good! 

4
 Very good. You win (3). (Win footnote.)
 

5
 This is where you should end up when you follow the named reference from the starting section. From here you can go to the second named section (4) or to the good end (3). 

